customer-orders/init-script.sql

```diff
@@ -0,0 +1,41 @@
+-- Create TABLE
+CREATE DATABASE customer_order;
+USE customer_order;
```

Comment:
Don't update the bal persist generated files. Update the docker compose file so that it points to the table and the DB